app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -22,7 +22,10 @@
       <ImageView
```

Comment:
Add `android:background="@color/white"` to ScrollView or the root layout

Reply:
Added.